index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+    <fieldset class="border">
+      <legend>Personal information</legend>
+
+      <label class="form-field">
+        Surname:
+        <input 
+        type="text" 
+        name="Surname"
+        autocomplete="off"><br>
+      </label>
+
+      <label class="form-field">
+        Name:
+        <input 
+        type="text" 
+        name="Name"
+        autocomplete="off"><br>
```

Comment:
```suggestion
        autocomplete="off"
      >
      <br>
```
Put tags on the new lines